sentry.client.config.ts

```diff
@@ -0,0 +1,12 @@
+import { init, Replay } from '@sentry/nextjs';
+
+import { SENTRY_DSN } from '@/next.constants.mjs';
+
+init({
+  dsn: SENTRY_DSN,
+  tracesSampleRate: 1,
+  debug: false,
+  replaysOnErrorSampleRate: 1.0,
+  replaysSessionSampleRate: 0.1,
+  integrations: [new Replay({ maskAllText: true, blockAllMedia: true })],
```

Comment:
We don't have any PII here - we should capture what is actually on the screen

https://docs.sentry.io/platforms/javascript/guides/nextjs/session-replay/privacy/

```suggestion
  integrations: [new Replay()],
```